Add ep 8 trailer full (#59)
psf/the-invisibles: content/english/episodes/ep-8-trailer.md

---
title: "Trailer: Episode 8 with Mojdeh Rastgoo"
meta_title: "Trailer: Episode 8 with Mojdeh Rastgoo"
description: "A preview of our chat with Mojdeh Rastgoo. Watch the full episode on April 15, 2025"
date: 2025-04-10T00:00:00-08:00
host: ["Tereza", "Georgi"]
draft: false
tags: ["python", "iran", "france", "pyladies", "psf-coc-wg", "pypodcats", "community", "project manager"]
publish_date: 2025-04-10T00:00:00-08:00
podcast_file: "/audio/hidden-figures-ep8-trailer.mp3"
podcast_duration: 98
episode_image: "images/ep8-mojdeh.png"
podcast_bytes: "3889257" # the length of the episode in bytes
images: ["images/ep8-mojdeh.png", "images/mojdeh.jpg"]
explicit: false 
type: 'episode'
episode: '8'
season: 1
transcript: '/transcript/hidden-figures-of-python-ep8-trailer.srt'
episode_type: trailer
layout: episode
youtube_id: "qetbNfnCkJM"
  

> speaker:

Reviewer: You need to add yourself to the "content/english/speakers" directory

  url: "/speakers/mojdeh-rastgoo/"
  title: "Mojdeh Rastgoo"

>   image: "/images/mojdeh_rastgoo.jpeg"

Reviewer: Is the image url correct? It doesn't show up in the deploy preview

Author: no indeed, I corrected it

  description: "

> Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Pris.

Reviewer: ```suggestion
Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Paris.
```

> She has a PhD in computer sicence and has been an active memnber of the community since 2018. 

Reviewer: ```suggestion
She has a PhD in Computer Science and has been an active member of the community since 2018. 
```

Reviewer: "of the community", should it say Python community, or open source community, perhaps?

> She is a co organizer of PyLadies Paris and member of PSF Code of Conduct Working-Group since mid 2023.

Reviewer: ```suggestion
She is a co-organizer of PyLadies Paris and member of PSF Code of Conduct Working Group since mid 2023.
```

And recently she has joined the PyPodcats team as the fifth organizer.  

> She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon.DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022. 

Reviewer: ```suggestion
She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022. 
```

Reviewer: Should be PyCon DE instead of PyCon.DE?

> She has also given introductory python tutorials in several occaisions.

Reviewer: ```suggestion
She has also given introductory Python tutorials in several occasions.
```

"
  social:
  - name: github
    icon: fa-brands fa-github
    link: https://github.com/mrastgoo

  - name: twitter
    icon: fa-brands fa-x-twitter
    link: https://x.com/RastgooMojdeh

  - name: linkedin
    icon: fa-brands fa-linkedin
    link: https://www.linkedin.com/in/mojdeh-rastgoo-6462753a/

  - name: web
    icon: fa-solid fa-globe
    link: https://mrastgoo.github.io./

---

> Sneak Peek of our chat with Mojdeh Rastgoo, hosted by Georgi Ker and Tereza Lofciu.

Reviewer: ```suggestion
Sneak Peek of our chat with Mojdeh Rastgoo, hosted by Georgi Ker and Tereza Iofciu.
```

> Since discovering Python and open source communities, Mojdeh has been actively involved in the python community, since she gave her first tutorial in 2018.

Reviewer: Maybe it would be better to split the sentence.

```suggestion
Since discovering Python and open source communities, Mojdeh has been actively involved in the Python community. In 2018, she gave her first tutorial at 
```
(and will in with where the tutorial was held)

> She is a member of PSF CoC working group, co organizer of PyLadies Paris and now PyPodcast host!!

Reviewer: ```suggestion
She is a member of PSF CoC Working Group, co-organizer of PyLadies Paris and now PyPodcast host!!
```

> In this chat, we chat about last year, introduce our new member Mojdeh, and talk about our plans for 2025.

Reviewer: ```suggestion
In this session, we shared a recap about our podcast series from last year. We also introduced our new member, Mojdeh, and talk about our plans for 2025.
```

---
title: "Trailer: Episode 8 with Mojdeh Rastgoo"
meta_title: "Trailer: Episode 8 with Mojdeh Rastgoo"
description: "A preview of our chat with Mojdeh Rastgoo. Watch the full episode on April 15, 2025"
date: 2025-04-10T00:00:00-08:00
host: ["Tereza", "Georgi"]
draft: false
tags: ["python", "iran", "france", "pyladies", "psf-coc-wg", "pypodcats", "community", "project manager"]
publish_date: 2025-04-10T00:00:00-08:00
podcast_file: "/audio/hidden-figures-ep8-trailer.mp3"
podcast_duration: 98
episode_image: "images/ep8-mojdeh.png"
podcast_bytes: "3889257" # the length of the episode in bytes
images: ["images/ep8-mojdeh.png", "images/mojdeh.jpg"]
explicit: false 
type: 'episode'
episode: '8'
season: 1
transcript: '/transcript/hidden-figures-of-python-ep8-trailer.srt'
episode_type: trailer
layout: episode
youtube_id: "qetbNfnCkJM"
  
speaker:
  url: "/speakers/mojdeh-rastgoo/"
  title: "Mojdeh Rastgoo"
  image: "/images/mojdeh.jpg"
  description: "
Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Paris.

> She has a PhD in Computer Science and has been an active member of the sicentific Python and open-source community since 2018.

Reviewer: ```suggestion
She has a PhD in Computer Science and has been an active member of the scientific Python and open-source community since 2018.
```
